tests/subsys/emds/emds_flash/src/main.c

```diff
@@ -21,661 +22,502 @@
 #if defined CONFIG_SOC_FLASH_NRF_RRAM
 #define RRAM DT_INST(0, soc_nv_flash)
 #define EMDS_FLASH_BLOCK_SIZE DT_PROP(RRAM, write_block_size)
-#define EXPECTED_STORE_TIME_BLOCK_SIZE (400)
-#define EXPECTED_STORE_TIME_1024 (6200)
+/* 32bits word in buffered stream is written typically 22us */
+/* 16 bytes should take ideally 88us (really takes 62 - 128us) */
+#define EXPECTED_STORE_TIME_BLOCK_SIZE (150)
+/* 1024 bytes should take ideally 5632us (really takes 5664 - 5824us) */
+#define EXPECTED_STORE_TIME_1024 (6000)
 #else
 #define FLASH DT_INST(0, soc_nv_flash)
 #define EMDS_FLASH_BLOCK_SIZE DT_PROP(FLASH, write_block_size)
-#define EXPECTED_STORE_TIME_BLOCK_SIZE (200)
-#define EXPECTED_STORE_TIME_1024 (13000)
+/* 32bits word is written typically 41us */
+#define EXPECTED_STORE_TIME_BLOCK_SIZE (50)
+/* 1024 bytes should take ideally 10496us (really takes 10895us) */
+#define EXPECTED_STORE_TIME_1024 (12000)
```

Comment:
It will be nice to compute this for platform specific defines and probably using arithmatic, so then we can change this later to quickly check different data sizes (like 2048 bytes, or 3000 bytes).

Reply:
Again, you are mixing emds api and emds flash. 
emds flash stores already prepared in advance data. Everything is already calculated in datasheet.
Timing calculation in api has not been implemented yet: https://github.com/nrfconnect/sdk-nrf/blob/collab-emds-next/subsys/emds/emds.c#L230-L233